package.json

```diff
@@ -40,9 +40,9 @@
     "clean": "rm -rf dist build node_modules stylesheets",
     "start-test-wiremock": "docker compose -f docker-compose-test.yml up -d",
     "generate-types": "script/generate-types",
-    "create-application:standard": "APPLICATION_TYPE='standard' npx playwright test --config ./e2e/playwright.config.ts --project=dev utils/createApplication.spec.ts",
-    "create-application:emergency": "APPLICATION_TYPE='emergency' npx playwright test --config ./e2e/playwright.config.ts --project=dev utils/createApplication.spec.ts",
-    "create-application:short-notice": "APPLICATION_TYPE='shortNotice' npx playwright test --config ./e2e/playwright.config.ts --project=dev utils/createApplication.spec.ts"
+    "create-application:standard": "APPLICATION_TYPE='standard' npx playwright test --config ./e2e/playwright.config.ts --project=local utils/createApplication.spec.ts",
```

Comment:
Has this crept in to the PR by accident? 
Seems useful though - so you can create an application using the e2e tests in local DB?